modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/computation/AbstractBulkComputation.java

```diff
@@ -102,17 +102,14 @@ public void processRecord(ComputationContext context, String inputStreamName, Re
             }
             delta.setProcessingEndTime(Instant.now());
             endBucket(context, delta);
-            context.askForCheckpoint();
         } else {
             if (isAbortedCommand(bucket.getCommandId())) {
                 log.debug("Skipping aborted command: {}", bucket.getCommandId());
-                context.askForCheckpoint();
             } else {
-                // this requires a manual intervention, the kv store might have been lost
-                throw new IllegalStateException(String.format("Unknown command: %s, offset: %s, record: %s.",
-                        bucket.getCommandId(), context.getLastOffset(), record));
+                log.warn("Skipping unknown command: {}, offset: {}.",bucket.getCommandId(), context.getLastOffset());
```

Comment:
Logging the record isn't interesting anymore?

Reply:
Not really, such record only contains a list of document ids and it is too verbose for a warning that could be produced in mass. With the offset, we can still dump the record if necessary.